planar/triangulate/incrementaldelaunaytriangulator.go

```diff
@@ -100,12 +100,12 @@ func (idt *IncrementalDelaunayTriangulator) InsertSite(v quadedge.Vertex) (*quad
 	// log.Printf("Made Edge: %v -> %v", base.Orig(), base.Dest());
 	quadedge.Splice(base, e)
 	startEdge := base
-	done := false
-	for !done {
+
+	for true {
 		base = idt.subdiv.Connect(e, base.Sym())
```

Comment:
change this to:
```
for {
```

For infinite loops in `go` you don't specify the conditional value.